server/migrations/20220227062712-create-book.cjs

```diff
@@ -18,7 +18,9 @@ module.exports = {
         type: Sequelize.INTEGER
       },
       cover: {
-        type: Sequelize.STRING
+        type: Sequelize.STRING,
+        defaultValue: null,
+        allowNull: true
```

Comment:
#L22-L23 are not necessary because the default is for attributes to be allowed null unless explicitly stated as `allowNull: false` or if the attribute is part of a unique index. The `defaultValue: null` directive has no effect.